wpilibj/src/main/java/edu/wpi/first/wpilibj/CAN.java

```diff
@@ -21,8 +22,8 @@
  * calls.
  */
 public class CAN implements Closeable {
-  public static final int kTeamManufacturer = 8;
-  public static final int kTeamDeviceType = 10;
+  public static final int kTeamManufacturer = CANAPITypes.CANManufacturer.kTeamUse.id;
+  public static final int kTeamDeviceType = CANAPITypes.CANDeviceType.kMiscellaneous.id;
```

Comment:
kTeamManufacturer and kTeamDeviceType seem a bit redundant now but I did not want to remove or change them as that would break api compatibility.